cmd/kusk/Makefile

```diff
@@ -41,20 +41,17 @@ ifeq ($(CONTAINER_ENGINE),)
 	CONTAINER_ENGINE = $(shell podman version >/dev/null 2>&1 && which podman)
 endif
 
-go_bindata := $(shell which go-bindata 2>/dev/null)
-ifeq ($(go_bindata),)
-go_bindata := cd hack/ci-deps && go install github.com/kevinburke/go-bindata/... && cd ../.. && "${GOPATH}/bin/go-bindata"
-endif
 
 .PHONY: all
 all: install-tools pre-commit
 
-kustomize: ## $(KUSTOMIZE) ## Download kustomize locally if necessary.
+build-tools: ## $(KUSTOMIZE) ## Download kustomize locally if necessary.
 # $(KUSTOMIZE): $(LOCALBIN)
 	mkdir -p $(shell pwd)/bin
 	rm -f $(KUSTOMIZE)
 	curl -s $(KUSTOMIZE_INSTALL_SCRIPT) | bash -s -- $(subst v,,$(KUSTOMIZE_VERSION)) $(LOCALBIN)
-
+	GOBIN=$(shell pwd)/bin go install -a -v github.com/go-bindata/go-bindata/...@latest
```

Comment:
@mbana is this good enough?

Reply:
Just do:

```sh
go install github.com/go-bindata/go-bindata/go-bindata@latest
```

Much easier.

Either way, I'm not that concerned.